test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildServerTestFixture.cs

```diff
@@ -14,6 +15,20 @@ public class BuildServerTestFixture : IDisposable
         public BuildServerTestFixture()
         {
             PipeName = Guid.NewGuid().ToString();
+
+#if DEBUG
+            var configuration = "Debug";
+#elif RELEASE
+            var configuration = "Release";
+#else
+#error Unknown Configuration
+#endif
+
+            var rzcDirectory = Path.GetFullPath($"../../../../../src/Microsoft.AspNetCore.Razor.Design/bin/{configuration}/netstandard2.0/tools/");
```

Comment:
Just use the sln location while loop that we do elsewhere instead of being super relative